README.md

```diff
@@ -458,38 +469,48 @@ Crates tailored for specific boards.
- ![crates.io](https://img.shields.io/crates/v/pygamer.svg)
-- [`trellis_m4`](https://crates.io/crates/trellis_m4) - Board support for the [NeoTrellis M4 board](https://www.adafruit.com/product/3938)
- ![crates.io](https://img.shields.io/crates/v/trellis_m4.svg)
-- [`feather-f405`](https://crates.io/crates/feather-f405) - Board support for the [Feather STM32F405 Express](https://www.adafruit.com/product/4382) ![crates.io](https://img.shields.io/crates/v/feather-f405.svg)
-- [`feather_m0`](https://crates.io/crates/feather_m0) - Board support for the [Feather M0 board](https://www.adafruit.com/product/2772) ![crates.io](https://img.shields.io/crates/v/feather_m0.svg)
-- [`feather_m4`](https://crates.io/crates/feather_m4) - Board support for the [Feather M4 board](https://www.adafruit.com/product/3857)
- ![crates.io](https://img.shields.io/crates/v/feather_m4.svg)
-- [`circuit_playground_express`](https://crates.io/crates/circuit_playground_express) - Board support for the [Circuit Playground Express board](https://www.adafruit.com/product/3333)
- ![crates.io](https://img.shields.io/crates/v/circuit_playground_express.svg)
-- [`edgebadge`](https://crates.io/crates/edgebadge) - Board support for the [EdgeBadge board](https://www.adafruit.com/product/4400)
- ![crates.io](https://img.shields.io/crates/v/edgebadge.svg)
-- [`gemma_m0`](https://crates.io/crates/gemma_m0) - Board support for the [Gemma M0 board](https://www.adafruit.com/product/3501)
- ![crates.io](https://img.shields.io/crates/v/gemma_m0.svg)
-- [`itsybitsy_m0`](https://crates.io/crates/itsybitsy_m0) - Board support for the [ItsyBitsy M0 board](https://www.adafruit.com/product/3727)
- ![crates.io](https://img.shields.io/crates/v/itsybitsy_m0.svg)
-- [`itsybitsy_m4`](https://crates.io/crates/itsybitsy_m4) - Board support for the [ItsyBitsy M4 Express board](https://www.adafruit.com/product/3800)
- ![crates.io](https://img.shields.io/crates/v/itsybitsy_m4.svg)
-- [`trinket_m0`](https://crates.io/crates/trinket_m0) - Board support for the [Trinket M0 board](https://www.adafruit.com/product/3500)
- ![crates.io](https://img.shields.io/crates/v/trinket_m0.svg)
-- [`adafruit-feather-rp2040`](https://github.com/rp-rs/rp-hal) - Board Support Crate for the [Adafruit Feather RP2040].
-- [`adafruit-itsy-bitsy-rp2040`](https://github.com/rp-rs/rp-hal) - Board Support Crate for the [Adafruit ItsyBitsy RP2040].
-- [`adafruit-kb2040`](https://github.com/rp-rs/rp-hal) - Board Support Crate for the [Adafruit KB2040].
-- [`adafruit-macropad`](https://github.com/rp-rs/rp-hal) - Board Support Crate for the [Adafruit Macropad].
-- [`adafruit-qt-py-rp2040`](https://github.com/rp-rs/rp-hal) - Board Support Crate for the [Adafruit QT Py RP2040].
-
+- [`metro_m0`](https://crates.io/crates/metro_m0) - Board support for the [Metro M0 board] in the [atsamd-rs] repo. It is a [atsamd-rs tier 1 support] support board. ![crates.io](https://img.shields.io/crates/v/metro_m0.svg)
```

Comment:
The word `support` is duplicated. Probably some copy&paste error.

Reply:
Thanks! fixed up in #358